library/entropy_poll.c

```diff
@@ -109,6 +109,20 @@ static int getrandom_wrapper( void *buf, size_t buflen, unsigned int flags )
 #endif /* SYS_getrandom */
 #endif /* __linux__ || __midipix__ */
 
+#if defined(__FreeBSD__) || defined(__DragonFly__)
+#include <sys/param.h>
+#if (defined(__FreeBSD__) && __FreeBSD_version >= 1200000) || \
+    (defined(__DragonFly__) && __DragonFly_version >= 500700)
+#include <errno.h>
+#include <sys/random.h>
+#define HAVE_GETRANDOM
+static int getrandom_wrapper( void *buf, size_t buflen, unsigned int flags )
+{
+    return getrandom( buf, buflen, flags );
+}
+#endif
+#endif
```

Comment:
Thanks for the quick update @devnexen.

Unfortunately your latest commit uses a tab instead of a space (L#124) and so has failed the CI. Can you please use spaces to indent instead and make sure that the beginning of L#124 lines up with L#123 as shown in the suggestion above.